client/my-sites/themes/logged-out.jsx

```diff
@@ -8,9 +8,7 @@ const ConnectedThemeShowcase = connectOptions( ThemeShowcase );
 
 export default ( props ) => (
 	<Main fullWidthLayout className="themes">
-		{ isEnabled( 'themes/showcase-i4/search-and-filter' ) && (
-			<BodySectionCssClass bodyClass={ [ 'is-section-themes-i4' ] } />
-		) }
+		<BodySectionCssClass section="themes" />
```

Comment:
IIRC the page already has the class name `is-section-themes` added, so we don't need to explicitly add it here. Could you confirm if this is the case?

Reply:
Thanks, you are right! Removed :)